ih-core/lib/steps/shell/default/10_engineering.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# This file wires in the scripts which are defined in the
+# ConsultingMD/engineering repo so that the scripts and functions
+# are available in your shell.
+
+ENGINEERING_DIR=${GR_HOME}/engineering
+ENGINEERING_BIN=${ENGINEERING_DIR}/bin
+ENGINEERING_BASH=${ENGINEERING_DIR}/bash
+
+if [[ -d "${ENGINEERING_BASH}" ]]; then
+
+  #Make sure engineering bin is in the path
+  if [[ ! "$PATH" =~ ${ENGINEERING_BIN} ]]; then
+    export PATH="${ENGINEERING_BIN}:${PATH}"
+  fi
+
+  # this sets up aws-environment, vault-token and more.
+  for f in "$ENGINEERING_BASH"/*; do
+    # shellcheck source=/dev/null
+    . "$f"
+  done
```

Comment:
Not directly related to this PR, but it might be good to audit what's in those scripts. I remember when I originally followed the onboarding instructions, these scripts overwrote a bunch of my customized bash config, and head-scratching ensued. I ended up cherry-picking the ones I wanted to source in my own bash initialization scripts.

Reply:
I agree, I'd like to put anything in engineering that does more than introduce a function behind a flag.